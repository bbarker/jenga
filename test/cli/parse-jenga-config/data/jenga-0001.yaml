submodule-dir: submods
mafia-lock: true
drop-deps: [ directory ]
