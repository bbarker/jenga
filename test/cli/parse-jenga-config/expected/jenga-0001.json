{
    "submodule-dir": "submods",
    "mafia-lock": true,
    "submodules": [],
    "drop-deps": [
        "directory"
    ]
}

